### PR TITLE
use simd-json-derive fixes #39

### LINKED ISSUE #39: use simd-json-derive for simd-json benchmarks
`simd-json` offers its own, optimized derive implementation, it would be nice to use that for the benchmarks instead of the serde ones.

I'll gladly make a PR but I didn't find where the datastructures are defined.

### PR DESCRIPTION
**Pull request by Licenser**
* update to latest simd-json version
* use simd-json-derive  for simd-json tests instead of serde-derive

closes #39

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -67,9 +71,10 @@ default = [
   "serde_bare",
   "serde_cbor",
   "serde_json",
-  "simd-json",
+  "simdjson",
   "speedy",
 ]
+simdjson = ["simd-json", "simd-json-derive"]
```
Comment (djkoloski) [inline Cargo.toml:77]: I think we can make a `simd-json` feature that enables `dep:simd-json` and `simd-json-derive` instead.

**Review by djkoloski — CHANGES_REQUESTED** [review]
One minor fix, this looks good otherwise.

File: Cargo.toml (hunk 1 above)
Comment (Licenser) [reply Cargo.toml:77]: 👍 I'll give tha ta shot :)

**Comment by Licenser** [comment]
I also re-set the auto-generated files as they shouldn't be changed in the PR.

**Comment by djkoloski** [comment]
I need to get around to fixing that workflow so it doesn't try to push. Thanks for the PR!

**Comment by Licenser** [comment]
Thanks for the merge :) the PR found a perf-bug in simd-json win win for doing it ;)

**Pull request merged**